services/initChatAnonymous/initChatAnonymousServices.ts

```diff
@@ -0,0 +1,70 @@
+import { AxiosResponse } from "axios";
+import apiAnonymous from "@shared/fetcherAnonymous";
+
+type User = {
+  user_id: string;
+  human_id: string;
+  country_code: string;
+  username: string;
+  real_name: null;
+  last_active_at: string;
+  profile_pic_path: string;
+  profile_pic_asset_id: string;
+  profile_pic_public_id: string;
+  status: string;
+  bio: string;
+  is_banned: boolean;
+  is_anonymous: boolean;
+  allow_anon_dm: boolean;
+  only_received_dm_from_user_following: boolean;
+  is_backdoor_user: boolean;
+  verified_status: string;
+  anon_user_info_color_code: string;
+  anon_user_info_color_name: string;
+  anon_user_info_emoji_code: string;
+  anon_user_info_emoji_name: string;
+};
+
+type Message = {
+  id: string;
+  text: string;
+  html: string;
+  type: string;
+  user: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `user` field in the `Message` type is currently typed as `any`. This could lead to potential issues with type safety. If possible, it would be better to define a specific type for this field.
```diff
-  user: any;
+  user: User; // or another appropriate type
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->